x/staking/keeper/historical_info_test.go

```diff
@@ -154,5 +159,6 @@ func TestGetAllHistoricalInfo(t *testing.T) {
 	}
 
 	infos := app.StakingKeeper.GetAllHistoricalInfo(ctx)
+	require.Equal(t, len(expHistInfos), len(infos))
```

Comment:
Interesting is this not checked in line 163?

Reply:
Yes, this check is not required.